Examples/MagicWeatherSwiftUI/Magic Weather SwiftUI.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -21,10 +21,10 @@
       },
       {
         "package": "Nimble",
-        "repositoryURL": "https://github.com/quick/nimble",
+        "repositoryURL": "https://github.com/Quick/Nimble.git",
```

Comment:
we use a couple of features in Nimble that aren't available in `main` yet, for matching against `Result`